mmv1/third_party/terraform/.copywrite.hcl.erb

```diff
@@ -0,0 +1,22 @@
+<% autogen_exception -%>
+schema_version = 1
+
+project {
+  license        = "MPL-2.0"
+  copyright_year = 2017
+
+  # (OPTIONAL) A list of globs that should not have copyright/license headers.
+  # Supports doublestar glob patterns for more flexibility in defining which
+  # files or folders should be ignored
+  header_ignore = [
+    # Some ignores here are not strictly needed, but protects us if we change the types of files we put in those folders
+    # See here for file extensions altered by copywrite CLI (all other extensions are ignored)
+    # Ignores below might not have an https://github.com/hashicorp/copywrite/blob/4af928579f5aa8f1dece9de1bb3098218903053d/addlicense/main.go#L357-L394
+    ".release/**",
+    ".changelog/**",
+    "examples/**",
+    "META.d/*.yml",
+    ".golangci.yml",
```

Comment:
This file has been floated as something to ignore. Need to confirm.

Reply:
I think we'll leave it up to teams to use their best judgement when it comes to ignoring some of these configurations, like `.golangci.yml`. It's definitely safe to ignore as long as it doesn't include any IP.